### ISSUE TITLE
SSL Handshake failure

### ISSUE THREAD

**Issue by xelawafs** [chemicstry/recksplorer#18]
I'm experiencing an ssl handshake failure when I run `node server.js --port 3002`: Terminal output:

```
==> 🌎 Listening on port 3002. Open up http://:::3002/ in your browser.
E0305 10:11:29.992363346   10356 ssl_transport_security.cc:187] ssl_info_callback: error occured.

E0305 10:11:29.993013794   10356 ssl_transport_security.cc:976] Handshake failed with fatal error SSL_ERROR_SSL: error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure.
{ Error: Connect Failed
    at ~/recksplorer/node_modules/grpc/src/client.js:554:15 code: 14, metadata: Metadata { _internal_repr: {} } }
~/recksplorer/server/lightning/lnd.js:28
            throw new Error(`LND connection failed. ${err}`);
            ^

Error: LND connection failed. Error: Connect Failed
    at Object.callback (~/recksplorer/server/lightning/lnd.js:28:19)
    at ~/recksplorer/node_modules/grpc/src/client.js:557:12
```

My LND `config`, `admin.macaroon` and `tlc.cert` are here: `~/.lnd`

**Comment by xelawafs** [comment]
I think this is related? https://github.com/grpc/grpc-node/issues/157

**Comment by chemicstry** [comment]
What version are you using? If it is after 5ef8bcf12c2b3edf411331599e47502f37e1afa2, then you have to use LND generated certificates. If it's before that commit, you have to generate new certificates as described here https://github.com/mably/lncli-web/blob/master/README.md#generate-lnd-certificates-compatible-with-nodejs-grpc

**Comment by chemicstry** [comment]
It was a problem with newer LND certs and grpc. Fixed in 4410f00604809cb150316623c037fb0c222d3a18
